db/fixtures/miq_product_features.yml

```diff
@@ -3913,6 +3913,10 @@
       :description: Check Compliance of Last Known Configuration
       :feature_type: control
       :identifier: container_node_check_compliance
+    - :name: Cockpit
+      :description: Launch Cockpit Console
+      :feature_type: control
+      :identifier: container_cockpit
```

Comment:
do we want separate RBAC features for container and a different one for VM or instance Cockpit button ? I'd say reuse the same feature.